src/utils/mapHtmlAttributesToReactElementAttributes.ts

```diff
@@ -4,19 +4,19 @@
  * SPDX-License-Identifier: AGPL-3.0-only
  */
 
-import { BOOLEAN_ATTRIBUTES } from '../dom/attributes/BooleanAttributes'
-import { REACT_ATTRIBUTES } from '../dom/attributes/ReactAttributes'
+import booleanAttributes from '../dom/attributes/booleanAttributes'
+import reactAttributes from '../dom/attributes/reactAttributes'
 import { isValidTagOrAttributeName } from './isValidTagOrAttributeName'
 
 /**
  * Returns the parsed attribute value taking into account things like boolean attributes
  *
- * @param {String} attribute The name of the attribute
- * @param {*} value The value of the attribute from the HTML
- * @returns {*} The parsed attribute value
+ * @param {string} attribute The name of the attribute
+ * @param {string} value The value of the attribute from the HTML
+ * @returns {string} The parsed attribute value
  */
-const getParsedAttributeValue = function (attribute: string, value: string) {
-  if (BOOLEAN_ATTRIBUTES.indexOf(attribute.toLowerCase()) >= 0) {
+function getParsedAttributeValue(attribute: string, value: string): string {
+  if (booleanAttributes.indexOf(attribute.toLowerCase()) >= 0) {
```

Comment:
```suggestion
  if (booleanAttributes.includes(attribute.toLowerCase())) {
```

Reply:
My original revision of this PR turned `booleanAttributes` into a Set (since that's what it is) – if you'd prefer using `includes` here, then I'll just go ahead and use `Set` and `Set#has` instead, since it has even broader compatibility than `includes` 👍

Reply:
https://caniuse.com/array-includes
https://node.green/#ES2016-features-Array-prototype-includes

I think using "includes" should be safe enough :sweat_smile:

Reply:
It was an easy enough change, and `.has` is approximately 2 times faster.